### DISCUSSION TITLE
can't figure out how to make it work. Need help.

### DISCUSSION THREAD

**Discussion by ANP-Oxy** [quarto-dev/quarto-nvim#56] [Q&A] [answered]
Hello, quarto-nvim community. I recently started using nvim and at first I was just using vim plugins but yesterday I switched to lua configuration by following a youtube playlist. I don't have much understanding of lua either, most of my config is copy pasted the chunks I thought were necessary, so I'm not really able to figure out what's going on. I use quarto on my windows os and want to use it on linux also so I wanted to add this plugin to my neovim configuration and I did add it but it's not working.  I tried looking at the files in neovim-kickstarter config but that didn't make sense to me either it's pretty advanced. 

I'm getting following error when I try to open a **.qmd** file. 
`error detected while processing BufReadPost Autocommands for "*.qmd":
Error executing lua callback: ...m/site/pack/packer/start/otter.nvim/lua/otter/keeper.lua:131: No query found for this file type
stack traceback:
        [C]: in function 'assert'
        ...m/site/pack/packer/start/otter.nvim/lua/otter/keeper.lua:131: in function 'activate'
        ...m/site/pack/packer/start/quarto-nvim/lua/quarto/init.lua:149: in function 'activate'
        ...m/site/pack/packer/start/quarto-nvim/lua/quarto/init.lua:162: in function <...m/site/pack/packer/start/quarto-nvim/lua/quarto/init.lua:160>`

[This is the link to my nvim configuration ](https://github.com/ANP-Oxy/nvim)

I have installed the required plugins using packer inside the plugins.lua file

use { "quarto-dev/quarto-nvim" } 
use { "jmbuhr/otter.nvim" }
use { 'nvim-treesitter/nvim-treesitter' }

Then I created a **quarto.lua** file
and put the gives configurations inside it
I'm not sure what I am doing wrong here.

**Comment by jmbuhr** [comment, marked as answer]
Which version of nvim are you on? I have a configuration for 0.8.3 and 0.9 (the latest and recommend) as linked in the readme.

I would also recommend to try the complete Kickstarter configuration at least once to confirm that it works before copy-pasting parts. You can also follow my playlist with it to see how the different parts work together.

**Reply by ANP-Oxy** [reply]
Yeah I suppose I will just do that. Thank you.
